### PR TITLE
[OSDOCS-8230]: Release note about agent-based hosted control planes

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-8230

Link to docs preview: https://66441--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues (scroll to the end of the "Known issues" section)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1900,6 +1900,8 @@ For more information about early access, xref:../updating/understanding_updates/
 
 * Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
 
+* In hosted control planes for {product-title}, on the bare metal platform, if a worker node fails, another node is not added to the hosted cluster, even when other worker nodes are available. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```
Comment (CrystalChun) [inline release_notes/ocp-4-14-release-notes.adoc:1903]:
```suggestion
* In hosted control planes for {product-title}, on the bare metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```

Small suggestion and added a note for a workaround. Feel free to adjust the language. Thanks!

**Review by CrystalChun — APPROVED** [review]
/lgtm

**Comment by lahinson** [comment]
@LiangquanLi930 PTAL at this release note for HyperShift when you can. Merge freeze for OCP 4.14 is next week. Thank you!

**Comment by kalexand-rh** [comment]
> @LiangquanLi930 PTAL at this release note for HyperShift when you can. Merge freeze for OCP 4.14 is next week. Thank you!

@LiangquanLi930, please confirm this release note.

**Review by bburt-rh — COMMENTED** [review]
I've added one suggestion and a nitpick. Lmk if you have questions. Thanks!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1900,6 +1900,8 @@ For more information about early access, xref:../updating/understanding_updates/
 
 * Currently, users cannot modify the `interface-specific` safe sysctl list by updating the `cni-sysctl-allowlist` config map in the `openshift-multus` namespace. As a workaround, you can modify, either manually or with a Daemon Set, the file `/etc/cni/tuning/allowlist.conf` on the node or nodes. (link:https://issues.redhat.com/browse/OCPBUGS-11046[*OCPBUGS-11046*])
 
+* In hosted control planes for {product-title}, on the bare metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine that is associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:1903]:
```suggestion
* In hosted control planes for {product-title}, on the bare-metal platform, if a worker node fails, another node is not automatically added to the hosted cluster, even when other agents are available. As a workaround, manually delete the machine that is associated with the failed worker node. (link:https://issues.redhat.com/browse/MGMT-15939[*MGMT-15939*])
```
See SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_b

**Comment by lahinson** [comment]
Thanks @bburt-rh and @tmalove! It looks like you reviewed at the same time and provided very similar feedback 😆 . I'll incorporate your suggestions. I appreciate it!

**Pull request merged**